src/main/java/servlets/BillingConfig.java

```diff
@@ -15,47 +15,54 @@
 
 import data.JsonConverter;
 import data.Vendor;
-import data.Account;
+import util.FormIdNames;
+import static servlets.VendorServlet.updateSheets;
 
 import java.io.IOException;
 import java.security.GeneralSecurityException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import util.FormIdNames;
 
-import static servlets.VendorServlet.updateSheets;
 
 @WebServlet("/BillingConfig")
 public class BillingConfig extends HttpServlet {
   private static final String CONTENT_TYPE_APPLICATION_JSON = "application/json;";
-  private static final String REDIRECT_READFILE = "/index.html";
-
-  private Vendor vendor;
-  private Account account;
+  private static final String REDIRECT_INDEX = "/index.html";
 
   /**
-   * Read data endpoint.
+   * Read data endpoint for both the edit and read pages.
+   *
    * @param request form that contains valid vendor and account IDs
+   *                required fields: String vendorID & boolean entireConfig
    * @param response returns a configuration in JSON format
    */
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String vendorID = request.getParameter(FormIdNames.VENDOR_ID);
     String accountID = request.getParameter(FormIdNames.ACCOUNT_ID);
+    boolean getEntireConfig = Boolean.parseBoolean(request.getParameter(
```

Comment:
This returns true or false for if it's the entire vendor config?

Reply:
yes